packages/css-data/src/property-parsers/translate.ts

```diff
@@ -0,0 +1,76 @@
+import * as csstree from "css-tree";
+import type {
+  InvalidValue,
+  TupleValue,
+  TupleValueItem,
+  Unit,
+} from "@webstudio-is/css-engine";
+import { cssTryParseValue } from "../parse-css-value";
+
+export const parseTranslate = (input: string): TupleValue | InvalidValue => {
+  let tokenStream = input.trim();
+  tokenStream = tokenStream.endsWith(";")
+    ? tokenStream.slice(0, -1)
+    : tokenStream;
+
+  const cleanupKeywords = ["translate:"];
+
+  for (const cleanupKeyword of cleanupKeywords) {
+    tokenStream = tokenStream.startsWith(cleanupKeyword)
+      ? tokenStream.slice(cleanupKeyword.length).trim()
+      : tokenStream;
+  }
+
+  const cssAst = cssTryParseValue(tokenStream);
+  if (cssAst === undefined) {
+    return {
+      type: "invalid",
+      value: input,
+    };
+  }
+
+  const parsed = csstree.lexer.matchProperty("translate", cssAst);
+  if (parsed.error) {
+    return {
+      type: "invalid",
+      value: input,
+    };
+  }
+
+  const translateValues: TupleValueItem[] = [];
+
+  try {
```

Comment:
is walking can result in runtime error?

Reply:
Nope, just comes in handy if we want to throw an error or semething while parsing. Just following from `filters`. Will remove, not actually needed here.